Makefile

```diff
@@ -1,8 +1,8 @@
 CSI_VERSION=1.1.0
 PROTOBUF_VERSION=3.11.2
 CURL=curl -Lsf
-KUBEBUILDER_VERSION = 2.0.1
-CTRLTOOLS_VERSION = 0.2.1
+KUBEBUILDER_VERSION = 2.2.0
+CTRLTOOLS_VERSION = 0.2.4
```

Comment:
It seems that the latest version of CTRLTOOL  is v0.2.5, but v0.2.4 is preferable?
https://github.com/kubernetes-sigs/controller-tools/releases

Reply:
controller-tools v0.2.5 depends on k8s 0.17, so skipped at this point.